### DISCUSSION TITLE
How to edit email templates and manage translations?

### DISCUSSION THREAD

**Discussion by gtomaka** [saleor/saleor#8743] [Q&A]
First of all, thank you so much for creating such a great product! Unfortunately, I have some issues with sending translated emails. 
I tried to follow Saleor 3.0 documentation (https://docs.saleor.io/docs/3.0/developer/running-saleor/emails). I cannot find `templates/templated_email`  and the method `npm run build-emails`  seems to not exist. Is this doc still relevant? How can I make it work? Is it possible to send emails to user based on his language? 

Cheers, Grzegorz

**Comment by krzysztofwolski** [comment]
Thank you for the reporting - linked page should be updated. In 3.0 email sending is handled by plugin https://docs.saleor.io/docs/3.0/developer/available-plugins/user-emails
